### PR TITLE
Added program area

### LINKED ISSUE #2270: Add Program Area: Brigade Organizers Playbook
### Overview
We need to add program area information to the Brigade Organizers Playbook project so that it will show up when we add the program areas filter.

### Action Items
- [x] Review the project.md file model in the Resources section
- [x] Add the following text to the file indicated in the Resources section
```
program-area:  
    - Civic Tech Infrastructure
```

### Resources/Instructions
File location to make the change: _projects/brigade-organizers-playbook.md

URL for UAT to check the page has been changed: https://github.com/hackforla/website/blob/gh-pages/_projects/brigade-organizers-playbook.md

[Model for project.md files](https://github.com/hackforla/website/wiki/project.md-file-template)

### PR DESCRIPTION
**Pull request by ghost**
Fixes #2270

### What changes did you make and why did you make them ?

  - added program area with value of civic tech infrastructure

### CODE REVIEW

**Comment by ghost** [comment]
Not sure if I was supposed to add an image of the changes. Let me know if I should fix that.

File: _projects/brigade-organizers-playbook.md [hunk 1]
```diff
@@ -6,6 +6,8 @@ image: /assets/images/projects/brigade-organizers-playbook.png
 alt: "Brigade Organizer's Playbook Extension Project"
 image-hero: /assets/images/projects/brigade-organizers-playbook-hero.png
 alt-hero: ''
+program-area:
```
Comment (SAUMILDHANKAR) [inline _projects/brigade-organizers-playbook.md:9]: @amnak1284: Hi Amna, thanks for working on this issue. Just wanted to share a quick note. As per Mathew's comment on the issue, "program area" text needs to be changed to "programs" as a result of "program areas" not being valid html attribute name due to spacing.

**Review by SAUMILDHANKAR — CHANGES_REQUESTED** [review]

File: _projects/brigade-organizers-playbook.md (hunk 1 above)
Comment (macho-catt) [reply _projects/brigade-organizers-playbook.md:9]: Actually, this should be fine as "program-area"
I changed the name to all of the properties and added the dash a month ago, so this should be fine. Thanks for looking out though @SAUMILDHANKAR

**Review by edeneault — APPROVED** [review]
Looks great!  Thank you for the contribution:)

**Review by SAUMILDHANKAR — APPROVED** [review]
Thanks Mathew for taking a look at this. 
I can see the changes in my environment and looks good to me. Thanks Amna.

**Pull request merged**